File: csharp/DVSClient/DVSClient/address/format/enrichment/LocationElements.cs
using DVSClient.Server.Address.Format;

namespace DVSClient.Address.Format.Enrichment
{
    public class LocationElements
    {
        public string Code { get; }
        public string Description { get; }
        public string SmallOrHomeOffice { get; }

        public LocationElements(RestApiEnrichmentLocationElements api)
        {
            Code = api.Code ?? string.Empty;
            Description = api.Description ?? string.Empty;
            SmallOrHomeOffice = api.SmallOrHomeOffice ?? string.Empty;
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/address/format/enrichment/LocationElements.java
package com.experian.dvs.client.address.format.enrichment;

import com.experian.dvs.client.server.address.format.RestApiEnrichmentLocationElements;

import java.util.Objects;

public class LocationElements {

    private final String code;
    private final String description;
    private final String smallOrHomeOffice;

    // Constructor
    public LocationElements(RestApiEnrichmentLocationElements api) {
        this.code = Objects.toString(api.getCode(), "");
        this.description = Objects.toString(api.getDescription(), "");
        this.smallOrHomeOffice = Objects.toString(api.getSmallOrHomeOffice(), "");
    }

    // Getters
    public String getCode() {
        return code;
    }

    public String getDescription() {
        return description;
    }

    public String getSmallOrHomeOffice() {
        return smallOrHomeOffice;
    }
}
